governance/tests/test-feishu-sync.sh

```
@@ -0,0 +1,300 @@
+DROPS=$(grep -c '"m": "DELETE", "p": "/open-apis/bitable/v1/apps/appSTUB/tables/tbl' "$CAP")
+[[ $RC -eq 0 && "$DROPS" -ge 1 ]] && pass "--drop 删表+单轮重建 exit 0" || fail "--drop（rc=$RC drops=$DROPS）"
+run_sync --verify; RC=$?
+[[ $RC -eq 0 ]] && pass "--drop 重建后 --verify 收敛（字段一致）" || fail "--drop 重建后未收敛 rc=$RC"
+
+# ---- 负向/守卫 ----
+out=$(FEISHU_SYNC_DISABLED=1 GH_TOKEN=x FEISHU_APP_ID=a FEISHU_APP_SECRET=s FEISHU_BITABLE_APP_TOKEN=t \
+  BUTLER_SHADOW_FILE="$FSHADOW" $FSYNC 2>&1); RC=$?
+[[ $RC -eq 0 && "$out" == *投影停用* ]] && pass "AC-7b 停用守卫=skipped 绿" || fail "停用守卫 rc=$RC"
+out=$(GH_TOKEN=x BUTLER_SHADOW_FILE="$FSHADOW" $FSYNC 2>&1); RC=$?
+[[ $RC -eq 0 && "$out" == *未配置* ]] && pass "凭据缺席=skipped 绿（过渡期）" || fail "凭据守卫 rc=$RC"
+GH_TOKEN= GH_API_BASE="$BASE" FEISHU_API_BASE="$BASE" FEISHU_APP_ID=a FEISHU_APP_SECRET=s \
```

Shellcheck (0.11.0)

[warning] Remove space after = if trying to assign a value (for empty string, use var='' ... ).

(SC1007)